brianlong/www.validators.app
# 188682313-bring-back-ping-times

#### What's this PR do?
- as in title
- user should be able to post data to collector endpoint containing ping stats to validators
- Collector Logic is used to process and save this data 
- it's accessible through ping_times API endpoint

#### How should this be manually tested?
- run tests
- start sidekiq
- try pushing new ping times data through collector api endpoint:
`localhost:3000/api/v1/collector`
example payload: `[{"user_id":1,"network":"testnet","from_account":"ABCD","from_ip":"123.123.123.123","to_account":"EFGH","to_ip":"255.255.255.255","min_ms":0.881,"avg_ms":1.238,"max_ms":2.428,"mdev":0.597},{"user_id":1,"network":"testnet","from_account":"ABCD","from_ip":"123.123.123.123","to_account":"IJKL","to_ip":"10.10.10.10","min_ms":1.881,"avg_ms":2.238,"max_ms":3.428,"mdev":1.597}]`
payload_type: "ping_times"
payload_version: 1
- check if old collector objects are destroyed

#### What are the relevant tickets?
- URL to the PivotalTracker ticket

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
